app/services/atmosphere/vm_updater.rb

```diff
@@ -45,7 +45,7 @@ def updated_at
     end
 
     def old_enough?
-      server.created < Atmosphere.childhood_age.seconds.ago
+      (vm.created_at || server.created) < Atmosphere.childhood_age.seconds.ago
```

Comment:
Discovery mechanism is not affected here. Please notice that `created_at` is `nil` for not new Active Record element - then `server.created` will be taken into account.